pieman.sh

```diff
@@ -60,6 +60,8 @@ INCLUDES=${INCLUDES:=""}
 
 OS=${OS:="raspbian-stretch-armhf"}
 
+CUSTOM_FILES=${CUSTOM_FILES:=""}
```

Comment:
The environment variables are arranged in alphabetical order. Please, stick to the rule.